plugins/packages/packages_inspector.rb

```diff
@@ -90,12 +90,22 @@ def inspect_dpkg
       )
     end
     packages.each_slice(100) do |packages_slice|
-      apt_cache_output = @system.run_command(
-        "apt-cache",
-        "show",
-        *packages_slice.map { |p| "#{p.name}=#{[p.version, p.release].compact.join("-")}" },
-        stdout: :capture
-      )
+      begin
+        apt_cache_output = @system.run_command(
+          "apt-cache",
+          "show",
+          *packages_slice.map { |p| "#{p.name}=#{[p.version, p.release].compact.join("-")}" },
+          stdout: :capture
+        )
+      rescue
+        # Older Debian systems do not support the version parameter for `apt-cache show`
+        apt_cache_output = @system.run_command(
+          "apt-cache",
+          "show",
+          *packages_slice.map(&:name),
+          stdout: :capture
```

Comment:
It worked for my Debian inspection and the alternative would be no results to my knowledge.